package-lock.json

```diff
@@ -52,7 +52,7 @@
         "react-custom-scrollbars": "^4.1.2",
         "sass": "^1.69.4",
         "mockdate": "^3.0.5",
-        "data-transparency-ui": "github:fedspendingtransparency/data-transparency-ui#v7.0.3",
+        "data-transparency-ui": "github:fedspendingtransparency/data-transparency-ui#v7.0.4",
```

Comment:
you may have to run npm install data-transparency-ui bc the other spot in the file hasn't updated
![image](https://github.com/user-attachments/assets/2ba76a0c-8a7a-4634-9dd0-72e8042638f1)